mass-surveillance-snippet/snippet.html

```diff
@@ -291,13 +289,13 @@
NjMC4yOTksMC41MTksMC4xMjEsMS4xODItMC4zOTcsMS40ODENCgkJCQlDODQuMzk5LDg0LjM3LDg0LjIxMyw4NC40MTcsODQuMDI5LDg0LjQxN3oiLz4NCgkJPC9nPg0KCQk8Zz4NCgkJCTxwYXRoIGZpbGw9IiM0RDRFNTMiIGQ9Ik05Mi45NDQsNzkuMTk4Yy0wLjI3NywwLTAuNTU2LTAuMTA2LTAuNzY3LTAuMzE4bC0zLjU4Ni0zLjU4OGMtMC40MjQtMC40MjQtMC40MjQtMS4xMSwwLTEuNTM0DQoJCQkJYzAuNDI0LTAuNDIzLDEuMTExLTAuNDIzLDEuNTMzLDBsMy41ODYsMy41ODhjMC40MjQsMC40MjQsMC40MjQsMS4xMSwwLDEuNTM0QzkzLjQ5OSw3OS4wOTIsOTMuMjIyLDc5LjE5OCw5Mi45NDQsNzkuMTk4eiIvPg0KCQk8L2c+DQoJPC9nPg0KPC9nPg0KPGc+DQoJPGc+DQoJCTxwYXRoIGZpbGw9IiM0RDRFNTMiIGQ9Ik02MS43MjksMTguOTE1Yy0wLjU5OSwwLTEuMDg0LTAuNDg1LTEuMDg0LTEuMDg0di01LjA3MWMwLTAuNTk5LDAuNDg1LTEuMDg0LDEuMDg0LTEuMDg0DQoJCQlzMS4wODQsMC40ODUsMS4wODQsMS4wODR2NS4wNzFDNjIuODEyLDE4LjQyOSw2Mi4zMjcsMTguOTE1LDYxLjcyOSwxOC45MTV6Ii8+DQoJPC9nPg0KCTxnPg0KCQk8cGF0aCBmaWxsPSIjNEQ0RTUzIiBkPSJNNTIuMDYsMTkuOTkxYy0wLjQ3OSwwLTAuOTE3LTAuMzItMS4wNDctMC44MDVsLTEuMzEyLTQuODk5Yy0wLjE1NC0wLjU3OSwwLjE4OC0xLjE3MywwLjc2OC0xLjMyOA0KCQkJYzAuNTc4LTAuMTU2LDEuMTczLDAuMTg4LDEuMzI4LDAuNzY3bDEuMzEyLDQuODk5YzAuMTU0LDAuNTc5LTAuMTg4LDEuMTczLTAuNzY4LDEuMzI4QzUyLjI0NywxOS45NzksNTIuMTUyLDE5Ljk5MSw1Mi4wNiwxOS45OTENCgkJCXoiLz4NCgk8L2c+DQoJPGc+DQoJCTxwYXRoIGZpbGw9IiM0RDRFNTMiIGQ9Ik00Mi4zNzYsMjMuMjQyYy0wLjM3NSwwLTAuNzM5LTAuMTk0LTAuOTQtMC41NDJsLTIuNTM0LTQuMzkzYy0wLjI5OS0wLjUxOS0wLjEyMS0xLjE4MiwwLjM5Ny0xLjQ4MQ0KCQkJYzAuNTIxLTAuMjk5LDEuMTgyLTAuMTIxLDEuNDgxLDAuMzk3bDIuNTM0LDQuMzkzYzAuMjk5LDAuNTE5LDAuMTIxLDEuMTgyLTAuMzk3LDEuNDgxQzQyLjc0NiwyMy4xOTUsNDIuNTYsMjMuMjQyLDQyLjM3NiwyMy4yNDINCgkJCXoiLz4NCgk8L2c+DQoJPGc+DQoJCTxwYXRoIGZpbGw9IiM0RDRFNTMiIGQ9Ik0zNC41MSwyNy42NTZjLTAuMjc3LDAtMC41NTYtMC4xMDYtMC43NjgtMC4zMThsLTMuNTg0LTMuNTg3Yy0wLjQyMy0wLjQyNC0wLjQyMy0xLjExLDAuMDAxLTEuNTM0DQoJCQljMC40MjQtMC40MjMsMS4xMS0wLjQyMywxLjUzNCwwbDMuNTg0LDMuNTg3YzAuNDIzLDAuNDI0LDAuNDIzLDEuMTEtMC4wMDEsMS41MzRDMzUuMDY0LDI3LjU1LDM0Ljc4NywyNy42NTYsMzQuNTEsMjcuNjU2eiIvPg0KCTwvZz4NCgk8Zz4NCgkJPGc+DQoJCQk8cGF0aCBmaWxsPSIjNEQ0RTUzIiBkPSJNNzEuODExLDE5Ljk5MWMtMC4wOTMsMC0wLjE4OC0wLjAxMi0wLjI4MS0wLjAzOGMtMC41NzktMC4xNTUtMC45MjItMC43NS0wLjc2OC0xLjMyOGwxLjMxMi00Ljg5OQ0KCQkJCWMwLjE1NS0wLjU3OSwwLjc1MS0wLjkyMiwxLjMyOC0wLjc2N2MwLjU3OSwwLjE1NSwwLjkyMiwwLjc1LDAuNzY4LDEuMzI4bC0xLjMxMiw0Ljg5OQ0KCQkJCUM3Mi43MjgsMTkuNjcxLDcyLjI4OSwxOS45OTEsNzEuODExLDE5Ljk5MXoiLz4NCgkJPC9nPg0KCQk8Zz4NCgkJCTxwYXRoIGZpbGw9IiM0RDRFNTMiIGQ9Ik04MS40OTIsMjMuMjQyYy0wLjE4NCwwLTAuMzctMC4wNDctMC41NDEtMC4xNDVjLTAuNTE5LTAuMy0wLjY5Ni0wLjk2My0wLjM5Ny0xLjQ4MWwyLjUzNS00LjM5Mw0KCQkJCWMwLjMtMC41MTksMC45NjItMC42OTYsMS40ODEtMC4zOTdjMC41MTksMC4zLDAuNjk2LDAuOTYzLDAuMzk3LDEuNDgxbC0yLjUzNSw0LjM5M0M4Mi4yMzEsMjMuMDQ3LDgxLjg2NywyMy4yNDIsODEuNDkyLDIzLjI0MnoNCgkJCQkiLz4NCgkJPC9nPg0KCQk8Zz4NCgkJCTxwYXRoIGZpbGw9IiM0RDRFNTMiIGQ9Ik04OS4zNTgsMjcuNjU2Yy0wLjI3NywwLTAuNTU1LTAuMTA2LTAuNzY3LTAuMzE3Yy0wLjQyNC0wLjQyNC0wLjQyNC0xLjExLDAtMS41MzRsMy41ODYtMy41ODcNCgkJCQljMC40MjItMC40MjQsMS4xMDktMC40MjMsMS41MzMsMGMwLjQyNCwwLjQyNCwwLjQyNCwxLjExLDAsMS41MzRsLTMuNTg2LDMuNTg3Qzg5LjkxNCwyNy41NSw4OS42MzYsMjcuNjU2LDg5LjM1OCwyNy42NTZ6Ii8+DQoJCTwvZz4NCgk8L2c+DQo8L2c+DQo8L3N2Zz4NCg==">
```

Comment:
We need valid XML, images should close with `/>`. Fixed here https://github.com/mozilla/snippets/commit/ac70254136eec16c0fd9f2d9890eebb22220dcf0